Assets/Scripts/Animation/PopupInventoryItem.cs

```diff
@@ -1,105 +1,73 @@
 ﻿using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
 
 public class PopupInventoryItem : MonoBehaviour {
-    private static Image gameObjectImage;
-    private static Vector3 originalPos;
-    private static bool itemRendered = false;
+    private Image gameObjectImage;
```

Comment:
We don't need/want things to be static anymore, since we have an instance reference in Inventory.cs.